src/taxonomy/taxonomy-detail/TagListTable.jsx

```diff
@@ -0,0 +1,49 @@
+import {
+  DataTable,
+  TextFilter,
+} from '@edx/paragon';
+import Proptypes from 'prop-types';
+
+import { useTagListDataResponse, useTagListDataStatus } from '../api/hooks/selectors';
+
+const TagListTable = ({ taxonomyId }) => {
+  const useTagListData = () => {
+    const { isError, isFetched } = useTagListDataStatus(taxonomyId);
+    const tagList = useTagListDataResponse(taxonomyId);
+    return { isError, isFetched, tagList };
+  };
+
+  const { tagList } = useTagListData(taxonomyId);
+
+  if (!tagList || !tagList.results) {
+    return 'Loading...';
+  }
+
+  return (
+    <DataTable
+      isPaginated
```

Comment:
Missing a few pagination-related settings here (ref: [Paragon: DataTable](https://paragon-openedx.netlify.app/components/datatable/)):
```suggestion
      isPaginated
      manualPagination
      pageCount={tagList.numPages}
      fetchData={fetchData}
```

^ needs a `fetchData` method implemented to make the next/prev page buttons fetch the next page of data, see the example under [DataTable#paginated-selection](https://paragon-openedx.netlify.app/components/datatable/#backend-filtering-and-sorting).

Reply:
Done 3d6b0dfe53b817861038e2758aa405cc9b182066

Reply:
I haven't dug into figure out why, but when you hit the prev/next buttons on the table here, we hit both these REST APIs. But we should only be hitting the 2nd `/tags/` endpoint.

1. `/api/content_tagging/v1/taxonomies/<pk>/`
2. `/api/content_tagging/v1/taxonomies/<pk>/tags/`

Reply:
This is also related to `React-Query`. If we refocus the window or [other conditions](https://tanstack.com/query/v4/docs/react/guides/important-defaults), we get a fetch in the background. This is the default behavior. We could tune this application-wise (the same happens in the TaxonomyList) if we think this doesn't suit us.

I was not able to reproduce the issue here. Maybe a windows refocus (not prev/next) triggered the call?

Reply:
Yeah, don't worry about extra fetch calls seen, it's just checking if the server data has changed, so it can automatically refresh the UI if needed. If the data hasn't changed, nothing will happen.